osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/mapping/SsldConverter.java

```diff
@@ -181,4 +182,13 @@ public boolean equals(final Object obj) {
                 && Objects.equals(this.ssldRepository, ((SsldConverter) obj).ssldRepository);
     }
 
+    @Override
+    public boolean canConvert(final Type<?> sourceType, final Type<?> destinationType) {
```

Comment:
Is this `canConvert` update related to earlier changes with a `BaseDevice` - `Device` hierarchy that has been removed with the addition of `UpdatedDevice` (in which case it seems this would be no longer needed) or does it stand on its own?